README.md

```diff
@@ -1 +1,73 @@
-# DeploymentCenter
\ No newline at end of file
+# DeploymentCenter
+
+## Description
+This Application provides GUI for kubernetes cluster management. You can add multiple connections and switch between them.
+Available Functions:
+- Deployments
+  - Create
+  - List
+  - Read Basic Info
+  - Read Pods with their status and logs
+  - Read Deployment Containers and Env Variables
+  - Read Volumes
+  - Scale Pods Count
+  - Delete
+- Load Balancers
+  - Create
+  - List
+  - Read
+  - Delete
+- Volumes
+  - Create
+  - List
+
+In future:
+- CronJobs
+- Request Limits in Deployments
+- Modyfing Env Variables in Deployments
+- Modyfing IP Addresses and Ports in LBs
+- More complex and versatile AI solution
+
+## System Architecture
+
+## How to deploy this solution to your server?
+
+You can use docker-compose for example:
```

Comment:
Yes, there should be a comma after "for example" to improve readability and clarity. The corrected sentence would be:

You can use Docker Compose, for example: